ConTabs/Table.cs

```diff
@@ -110,6 +122,9 @@ private Table()
             TableStyle = Style.Default;
             HeaderAlignment = Alignment.Default;
             ColumnAlignment = Alignment.Default;
+            TableAlignment = Alignment.Default;
+            CanvasWidth = Console.WindowWidth - 1; // it would word wrap
```

Comment:
Binding to the System.Console class worries me. What happens when someone wants to use ConTabs outside of the console?

So far we've been able to ignore the Console by treating everything as a simple string and leaving it up to our downstream consumers to work out what to do with it. On the other hand, the system console is probably where ConTabs gets called 99% of the time, so there is merit in making some assumptions. It's a tricky one!

Two options spring to mind:

1. Use a new class (called e.g. `TableContainer`) that wraps a `Table` object and is coupled to Console with this sort of code. The the container could create the table, set the width based on the console width, and render it to the console directly.

2. Add a helper method somewhere that sets the `CanvasWidth` to `Console.WindowWidth - 1`.

3. Add some logic that tries to detect whether we're in a Console (maybe [like this?](https://stackoverflow.com/a/15268625/50151)) and set the `CanvasWidth` to the console width or a sensible default.

Each have their own advantages and disadvantages. But I think it is important to retain our "portability" - even if its only for the 1% of users who run ConTabs in more exotic contexts.

PS After a bit more thought, I think those three options are in my order of preference. Still keen to hear your thoughts though.